modules/role/manifests/st2.pp

```diff
@@ -22,4 +20,12 @@
       include ::profile::hubot
     }
   }
+
+  if $_enterprise_token {
+    include ::profile::st2flow
+    include ::profile::st2rbac
+    include ::profile::enterprise_auth_backend_ldap
+  } else {
+    include ::profile::auth_backend_pam
```

Comment:
This probably needs to be handled in the installer, tbh. We need to get info from the user (base DN, server info, user key).

Is it possible to run multiple auth backends? Then we can be additive versus selective.